app-datepicker.html

```diff
@@ -1077,7 +1077,7 @@
           if (_maxDateObj) {
             _isMoreThanMaxDate = _currentDate.getTime() > _maxDateObj.getTime();
           }
-          _isDisableDates = this.disableDates.some(function(date) {
+          _isDisableDates = !this.disableDates || this.disableDates.some(function(date) {
```

Comment:
This doesn't solve the issue yet every single date is disabled with this patch.

[See demo using your PR branch](http://jsbin.com/pabeligefo/edit?html,console,output)

I propose:

```js
// this.disableDates - to ensure disableDates is not null or undefined.
// !this.disableDates.length - to ensure disableDates's length not zero (since it's an array)
// this.disableDates.some - if disableDates passed all the above tests, proceed to the some function.
_isDisableDates = this.disableDates && !this.disableDates.length && this.disableDates.some(...);
```